arangod/Pregel/Conductor.cpp

```diff
@@ -840,42 +813,33 @@ void Conductor::finishedWorkerFinalize(VPackSlice data) {
   }
   receive(CleanupFinished{});
 
-  _timing.total.finish();
-
-  VPackBuilder debugOut;
-  debugOut.openObject();
-  debugOut.add("stats", VPackValue(VPackValueType::Object));
-  _statistics.serializeValues(debugOut);
-  debugOut.close();
-  _aggregators->serializeValues(debugOut);
-  debugOut.close();
-
-  LOG_PREGEL("063b5", INFO)
-      << "Done. We did " << _globalSuperstep << " rounds."
-      << (_timing.loading.hasStarted()
-              ? fmt::format("Startup time: {}s",
-                            _timing.loading.elapsedSeconds().count())
-              : "")
-      << (_timing.computation.hasStarted()
-              ? fmt::format(", computation time: {}s",
-                            _timing.computation.elapsedSeconds().count())
-              : "")
-      << (_storeResults ? fmt::format(", storage time: {}s",
-                                      _timing.storing.elapsedSeconds().count())
-                        : "")
-      << ", overall: " << _timing.total.elapsedSeconds().count() << "s"
-      << ", stats: " << debugOut.slice().toJson();
-
-  // always try to cleanup
-  if (_state == ExecutionState::CANCELED) {
-    auto* scheduler = SchedulerFeature::SCHEDULER;
-    if (scheduler) {
-      uint64_t exe = _executionNumber;
-      scheduler->queue(RequestLane::CLUSTER_AQL,
-                       [this, exe, self = shared_from_this()] {
-                         _feature.cleanupConductor(exe);
-                       });
-    }
+  if (_state != ExecutionState::CANCELED) {
```

Comment:
this whole block is deleted in one of the next PRs, here just make sure that this does not run when we are in the Canceled State